src/ZooKeeper/ZooKeeperCommon.h

```diff
@@ -565,9 +572,19 @@ struct ZooKeeperMultiRequest final : MultiRequest, ZooKeeperRequest
         std::for_each(requests.begin(), requests.end(), func);
         return base;
     }
+
+    enum class OperationType : UInt8
```

Comment:
It is better add a new type Unspecified and replace type of `operation_type` from `std::optional<OperationType>` to `OperationType`.